src/data-table/utils/selection.tsx

```diff
@@ -70,11 +70,11 @@ export function getSelectionMessages<T extends ReferenceObject>(
             _(row)
                 .pick(childrenKeys)
                 .values()
-                .flatten()
                 .value()
         )
-        .flatten()
+        .flattenDeep()
```

Comment:
[minor-subjective] When using `_.flattenDeep`, we lose a bit the perspective on the collection types we are working with. In this case, the inner flatten + outer flatMap should be equivalent.

Reply:
Guys, I am merging this PR as it is a dependency in this other [PR](https://github.com/EyeSeeTea/Bulk-Load/pull/75) that I have already merged. Feel free to create new PR if there is anything that worth to be changed!